src/components/serverPaginatedTable/serverPaginatedTable.js

```diff
@@ -159,6 +162,8 @@ class ServerPaginatedTableView extends React.Component {
 
     sortDirection = Object.keys(sortOrder).length === 0 ? this.props.defaultSortDirection || 'asc' : sortOrder.direction;
     sortColumn = Object.keys(sortOrder).length === 0 ? this.props.defaultSortCoulmn || '' : sortOrder.name;
+    window.sortColumnValue = sortColumn;
```

Comment:
The other way of approach is to create a new variable in the cart reducer to hold the sort field and a new function in the reducer to update the variable. Read the variable at the cart controller and make the default sort column. From cart view pass the reducer function to here and when the sort is updated update the cart reducer function which updates the cart sort value